lib/ex.coffee

```diff
@@ -199,9 +209,9 @@ class Ex
   substitute: (range, args) ->
     args = args.trimLeft()
     delim = args[0]
-    if /[a-z1-9\\"|]/i.test(delim)
+    if /[a-z]/i.test(delim)
```

Comment:
Huh, looks like you worked off of code that didn't include #102. Maybe try pulling the repository again?